dydx-testnet-4/pregenesis.json

```diff
@@ -1,10 +1,10 @@
 {
-  "genesis_time": "2023-10-11T04:19:58.475018Z",
+  "genesis_time": "2023-01-01T00:00:00Z",
```

Comment:
Change?

Reply:
This is intended with a fixed time in the past
 Testnet final genesis needs to be manually editted later